2.hello-kubernetes/node/package-lock.json

```diff
@@ -0,0 +1,410 @@
+{
+  "name": "node_server",
+  "version": "1.0.0",
+  "lockfileVersion": 1,
```

Comment:
I don't think we don't need to check this in.

Reply:
Deleted.